### PR TITLE
ETS-1471 age selector

### PR DESCRIPTION
**Pull request by cllagosr**

### CODE REVIEW

File: source/scss/components/_fti-select.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.fti-select {
+  padding: 0;
+
+  >span {
+    color: $gray-400;
+    font-size: 12px;
+    font-weight: bold;
+    margin-left: 15px;
```
Comment (BirgitPohl) [inline source/scss/components/_fti-select.scss]: Maybe you can change it in a future PR, in ETS-1382 we are introducing a variable that is using 15px for spacings, to keep them all together.

File: source/scss/fti-bootstrap/_custom-variables.scss [hunk 2]
```diff
@@ -40,10 +40,11 @@
 
 $gray-100: #f4f4f4;
 $gray-300: #b3b3b3;
+$gray-400: #929497;
 $gray-600: #666666;
 $gray-800: #424242;
 
-$red: #c01d1d;
+$red: #fc1119;
```
Comment (BirgitPohl) [inline source/scss/fti-bootstrap/_custom-variables.scss:47]: We are changing a red. Is this on purpose? Are all previous components ok with it?

**Review by BirgitPohl — DISMISSED** [review]

File: source/scss/components/_fti-select.scss (hunk 1 above)
Comment (JenyMzo) [reply source/scss/components/_fti-select.scss]: Sure :)

File: source/scss/style.scss [hunk 3]
```diff
@@ -48,6 +49,7 @@
 @import "components/text-image";
 @import "components/tab-menu";
 @import "components/dropdown";
+@import "components/fti-select";
```
Comment (brunogeronimo) [inline source/scss/style.scss]: Maybe we should rename it to `select`, since we are not using `fti` in any other component. What do you think?

**Review by brunogeronimo — DISMISSED** [review]

**Review by brunogeronimo — APPROVED** [review]

**Pull request merged**